src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaServiceTest.java

```diff
@@ -144,7 +143,7 @@ private void verifyTaskStateUpdateWasCalled(String taskId, TaskState newTaskStat
         );
     }
 
-    private List<CamundaVariableInstance> mockedVariablesResponse(String processInstanceId) {
+    private List<CamundaVariableInstance> mockedVariablesResponse(String processInstanceId, String taskId) {
```

Comment:
Maybe this comment belongs elsewhere and you have done it already.  Could be good idea to maintain a test where you search using processVariables and check what we get back.  So in theory these should stay the same and never be updated during a process.

Reply:
good idea I will raise another PR and use the /assign logic to update the task id and assert that the taskState processVariable should remain unchanged.